src/Logging/InitializationLogger.cs

```diff
@@ -24,7 +24,15 @@ public static class InitializationLogger
 
 		private static ILoggingBuilder LoadInitializationLogger(this ILoggingBuilder builder)
 		{
-			builder.AddConsole();
+			// Check if we are running on Service Fabric, if we are not on Service Fabric, add console logging
+			string sfAppName = Environment.GetEnvironmentVariable("Fabric_ApplicationName");
```

Comment:
I think nullability is enabled in these libs, so you have to use explicitly nullable types.